src/screens/OrganizationEvents/OrganizationEvents.tsx

```diff
@@ -83,41 +83,53 @@ function organizationEvents(): JSX.Element {
     e: ChangeEvent<HTMLFormElement>
   ): Promise<void> => {
     e.preventDefault();
-    try {
-      const { data: createEventData } = await create({
-        variables: {
-          title: formState.title,
-          description: formState.eventdescrip,
-          isPublic: publicchecked,
-          recurring: recurringchecked,
-          isRegisterable: registrablechecked,
-          organizationId: currentUrl,
-          startDate: dayjs(startDate).format('YYYY-MM-DD'),
-          endDate: dayjs(endDate).format('YYYY-MM-DD'),
-          allDay: alldaychecked,
-          location: formState.location,
-          startTime: !alldaychecked ? formState.startTime + 'Z' : null,
-          endTime: !alldaychecked ? formState.endTime + 'Z' : null,
-        },
-      });
-
-      /* istanbul ignore next */
-      if (createEventData) {
-        toast.success(t('eventCreated'));
-        refetch();
-        hideInviteModal();
-        setFormState({
-          title: '',
-          eventdescrip: '',
-          date: '',
-          location: '',
-          startTime: '08:00:00',
-          endTime: '18:00:00',
+    if (
+      formState.title.trim().length > 0 &&
+      formState.eventdescrip.trim().length > 0 &&
+      formState.location.trim().length > 0
+    ) {
+      try {
+        const { data: createEventData } = await create({
+          variables: {
+            title: formState.title,
+            description: formState.eventdescrip,
+            isPublic: publicchecked,
+            recurring: recurringchecked,
+            isRegisterable: registrablechecked,
+            organizationId: currentUrl,
+            startDate: dayjs(startDate).format('YYYY-MM-DD'),
+            endDate: dayjs(endDate).format('YYYY-MM-DD'),
+            allDay: alldaychecked,
+            location: formState.location,
+            startTime: !alldaychecked ? formState.startTime + 'Z' : null,
+            endTime: !alldaychecked ? formState.endTime + 'Z' : null,
+          },
         });
+
+        /* istanbul ignore next */
+        if (createEventData) {
+          toast.success(t('eventCreated'));
+          refetch();
+          hideInviteModal();
+          setFormState({
+            title: '',
+            eventdescrip: '',
+            date: '',
+            location: '',
+            startTime: '08:00:00',
+            endTime: '18:00:00',
+          });
+        }
+      } catch (error: any) {
+        /* istanbul ignore next */
+        errorHandler(t, error);
       }
-    } catch (error: any) {
-      /* istanbul ignore next */
-      errorHandler(t, error);
+    } else if (formState.title.trim().length === 0) {
+      toast.warning('Please enter a valid title!');
```

Comment:
Remove these error messages with more descriptive one's that tell the user that the same can't be blank.

Reply:
I changed the error messages. as `Title can not be blank`, `Description can not be blank` and `Location can not be blank`